app/assets/stylesheets/components/_super-navigation-header.scss

```diff
@@ -50,12 +84,16 @@ $breakpoint: 769px;
   }
 }
 
+.app-c-super-navigation-header__item-icon {
+  display: none;
+}
+
 // Search
 
 .app-c-super-navigation-header__item--search {
```

Comment:
Not a big deal and noted so that we're aware - when the full-width bar is not being used, the blue of the search toggle / link and the blue of the bar don't match:

![image](https://user-images.githubusercontent.com/1732331/122945801-bcaa3000-d370-11eb-9373-99abbd01f29d.png)

This is because the blue bar's colour comes from the deprecated `govuk_layout`, which uses the old blue - as `govuk_layout` gets removed this issue will be fixed.